cli/azd/cmd/up.go

```diff
@@ -23,6 +23,22 @@ func upCmd(rootOptions *commands.GlobalCommandOptions) *cobra.Command {
 			&infraCreateAction{
 				rootOptions: rootOptions,
 			},
+			// Print an additional newline to separate provision from deploy
```

Comment:
This is pretty clever - I would have never thought of this before - but seeing it now I think it's a very cute trick.